src/frontend/shortlist/src/views/VerifiedView.vue

```diff
@@ -0,0 +1,88 @@
+<script>
+import axios from "axios";
+import { useRoute } from "vue-router";
+export default {
+  data() {
+    return {
+      isVerified: false,
+      email: "",
+      errorMessage: "",
+      testVerified: false,
+      testNotVerified: false,
+      testPostSent: false,
+    };
+  },
+  setup() {
+    return { router: useRoute() };
+  },
+  mounted() {
+    //fetching information from url sent to the email address
+    let params = this.router.query;
+    this.isVerified = params.verified;
+    this.errorMessage = params.message;
+    //console.log(this.email);
+    //console.log(this.errorMessage);
+  },
+  methods: {
+    resendLink() {
+      //console.log(this.router.query);
+      axios
+        //link needs update after deployment
+        .post("http://127.0.0.1:8000/auth/request-resend-email", {
```

Comment:
I'll let this go for now, but if this whole axios call was moved into the javascript API (shortlist.js), then you wouldn't have to hardcode this address here.

Reply:
Ah I was wondering what to put there, since later I will need to change it after deployment. Lemme check out shortlist.js